handler/src/main/java/io/netty/handler/stream/ChunkedStream.java

```diff
@@ -122,7 +122,11 @@ public ByteBuf readChunk(ByteBufAllocator allocator) throws Exception {
         ByteBuf buffer = allocator.buffer(chunkSize);
         try {
             // transfer to buffer
-            offset += buffer.writeBytes(in, chunkSize);
+            int written = buffer.writeBytes(in, chunkSize);
```

Comment:
This behaviour could also be made more clear in the javadocs to `writeBytes`. The equivalent `setBytes` calls it out.

Reply:
Will do a followup PR for that.